docs/tutorials/cloud-agent.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Run an agent and flows in the cloud with containers.
+tags:
+    - Docker
+    - containers
+    - agents
+    - cloud
+    - tutorial
+---
+
+# Quickstart: Run an Agent in the Cloud
+
+## Run an Agent with Azure Container Instances
+
+Microsoft Azure Container Instances (ACI) provides a convenient and simple service for quickly spinning up a Docker container that can host a Prefect Agent and execute flow runs for deployments.
+
+## Prerequisites
+
+To follow this quickstart, you'll need the following:
+
+- A [Prefect Cloud account](/ui/cloud-quickstart/)
+- A Prefect Cloud [API key](/ui/cloud-api-keys/) (Prefect Cloud organizations may use a [service account](/ui/service-accounts/) API key)
+- A [storage block](/concepts/storage/) for storing deployed flow code (see the [Storage and Infrastructure tutorial](/tutorials/storage/) for instructions)
+- A [Microsoft Azure account](https://portal.azure.com/)
+- Azure CLI [installed](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) and [authenticated](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli)
+
+## Create a resource group
+
+Like most Azure resources, ACI applications must live in a resource group. If you don’t already have a resource group you’d like to use, create a new one by running the `az group create` command. For example, this example creates a resource group called `prefect-agents` in the `eastus` region:
+
+<div class='terminal'>
+```bash
+az group create --name prefect-agents --location eastus
+```
+</div>
+
+Feel free to change the group name or location to match your use case. You can also run `az account list-locations -o table` to see all available resource group locations for your account.
+
+## Create the container instance
+
+Prefect provides [pre-configured Docker images](/concepts/infrastructure/#docker-images) you can use to quickly stand up a container instance. These Docker images include Python and Prefect. For example, the image `prefecthq/prefect:2-python3.10` includes the latest release version of Prefect and Python 3.10.
+
+To create the container instance, use the `az container create` command. This example shows the syntax, but you'll need to provide the correct values for `[ACCOUNT-ID]`,`[WORKSPACE-ID]`, `[API-KEY]`, and any dependencies you need to `pip install` on the instance. These options are discussed below.
+
+<div class='terminal'>
+```bash
+az container create \
+--resource-group prefect-agents \
+--name prefect-agent-example \
+--image prefecthq/prefect:2-python3.10 \
+--secure-environment-variables PREFECT_API_URL='https://api.prefect.cloud/api/accounts/[ACCOUNT-ID]/workspaces/[WORKSPACE-ID]' PREFECT_API_KEY='[API-KEY]' \
+--command-line "/bin/bash -c 'pip install adlfs s3fs requests pandas; prefect agent start -p default-agent-pool -q test'"
```

Comment:
We could set the `EXTRA_PIP_PACKAGES` env var to shorten the command.

Reply:
Thanks for the heads up. I'll look into adding this. I think showing both methods might be good?

Reply:
`EXTRA_PIP_PACKAGES` doesn't work for containers created directly on ACI.